src/briefcase/integrations/android_sdk.py

```diff
@@ -1342,7 +1354,7 @@ def start_app(self, package, activity, passthrough):
                 f"Unable to start {package}/{activity} on {self.device}"
             ) from e
 
-    def logcat(self, pid):
+    def logcat(self, pid: str):
```

Comment:
Based on the current implementation, it must be a string because `subprocess.Popen()` does not accept `int`s in the command line to pass to the OS.

We can coerce it to string.